src/packages/console/src/components/sectionChooserDialog.tsx

```diff
@@ -19,23 +18,23 @@ export interface ISectionChooserDialogProps {
 export const SectionChooserDialog = (props: ISectionChooserDialogProps): React.ReactElement => {
   const { everypageClient } = useGlobals();
   const [sectionCategories, setSectionCategories] = React.useState<SectionCategory[] | undefined>(undefined);
-  const [selectedSectionCategoryId, setSelectedSectionCategoryId] = React.useState<number | undefined>(undefined);
+  const [selectedSectionCategoryId, setSelectedSectionCategoryId] = React.useState<string | undefined>(undefined);
```

Comment:
its better to store this as a number and convert to string when needed, not the other way around